### PR TITLE
Folders, Notepad and Book Icon Updates and new props

### PR DESCRIPTION
**Pull request by AmoryBlaine**
- New folders, notepad and book sprites
- New UPP book props
- Changes all old book icons to be new book icons
-  Deletes old book icons

### CODE REVIEW

File: code/game/objects/prop.dm [hunk 1]
```diff
@@ -302,6 +302,28 @@
...
 
+/obj/item/prop/magazine/book/uppbooklet
+	name = "UPP Party Doctrine Booklet"
+	desc = "A booklet provided to every single citizen of the Union of Progressive Peoples. The first page is stamped with the Party's insignia followed by a emphatic dedication to the working class; past, present, and future. Aside from being an easily digestible summary of the collective ideology of the Union, the booklet provides you with an outline of the Union's history and a good chunk is dedicated to listing and summarizing each of the Union's many member states."
+	icon_state = "book_upp"
+
+/obj/item/prop/magazine/book/warisaracket
+	name = "War is a Racket"
+	desc = "A book critical of the United States Marine Corps by Marine Corps legend, Major General Smedley Butler. In short, the book outlines the function which the historical Marine Corps had in establishing and enforcing American global hegemony. Good thing its critiques have no application contemporarily. Right?"
```
Comment (Doubleumc) [inline code/game/objects/prop.dm:312]:
```suggestion
	desc = "A book critical of the United States Marine Corps by Marine Corps legend, Major General Smedley Butler. In short, the book outlines the function which the historical Marine Corps had in establishing and enforcing American global hegemony. Good thing its critiques have no contemporary application. Right?"
```

**Review by Doubleumc — CHANGES_REQUESTED** [review]
Just a minor phrasing issue. Otherwise, code good.

**Review by Doubleumc — APPROVED** [review]

File: code/game/objects/prop.dm [hunk 2]
```diff
@@ -302,6 +302,28 @@
...
+	desc = "A book critical of the United States Marine Corps by Marine Corps legend, Major General Smedley Butler. In short, the book outlines the function which the historical Marine Corps had in establishing and enforcing American global hegemony. Good thing its critiques have no contemporary application. Right?"
+
+/obj/item/prop/magazine/book/littleredbook
+	name = "Quotations from Chairman Mao Tse-tung"
+	desc = "A tiny red book that fits in the palm of your hand. While not required reading, it provides the reader with easy access to- as the name implies- Quotations from Chairman Mao Tse-tung."
+	icon_state = "littleredbook"
+
+/obj/item/prop/magazine/book/communistmanifesto
+	name = "Communist Manifesto"
+	desc ="The bare minimum in communist reading."
```
Comment (AndroBetel) [inline code/game/objects/prop.dm:321]:
```suggestion
	desc = "The bare minimum in communist reading."
```

File: code/modules/paperwork/folders.dm [hunk 3]
```diff
@@ -26,6 +26,9 @@
 	desc = "A black folder."
 	icon_state = "folder_black"
 
+/obj/item/folder/green
+	desc = "A green folder."
+	icon_state = "folder_green"
```
Comment (AndroBetel) [inline code/modules/paperwork/folders.dm:31]:
```suggestion
	icon_state = "folder_green"

```

**Review by AndroBetel — APPROVED** [review]

**Pull request merged**